src/sentry/models/event.py

```diff
@@ -437,15 +437,12 @@ def get_minimal_user(self):
             email = self.snuba_data["email"]
             username = self.snuba_data["username"]
             ip_address = self.snuba_data["ip_address"]
-        else:
-            user_id = self.data["user_id"]
-            email = self.data["email"]
-            username = self.data["username"]
-            ip_address = self.data["ip_address"]
```

Comment:
I am not sure I get why this did actually not work.

Reply:
These properties are actually nested under `self.data["user"]` if they are present, this just gave a KeyError. The EventCommon implementation (`get_interface("user")` takes care of loading them properly.